### DISCUSSION TITLE
what support PHP version in peachpie ?

### DISCUSSION THREAD

**Discussion by xperia-query** [peachpiecompiler/peachpie#1119] [Q&A] [answered]
php 7.0 ~ 8.0 ?

or php 5.6 ~ php 8.0?

**Comment by bfistein** [comment, marked as answer]
PeachPie practically supports all versions of PHP up until 8.0. Note, in very old versions of PHP there could be some inconsistencies; the earliest version we tested or worked with was 5.6.
